sympy/combinatorics/perm_groups.py

```diff
@@ -1830,6 +1832,8 @@ def is_normal(self, gr, strict=True):
         for g1 in gens1:
             for g2 in gens2:
                 p = _af_rmuln(g1, g2, _af_invert(g1))
+                if all(i == p[i] for i in range(len(p))):
```

Comment:
There is  a method, `.is_Identity` for testing this. Moreover, it would be more efficient to test `g1`. It that is the identity, then `p` would be the identity for each `g2` and there is no need to loop over `gens2`.

Reply:
I'm aware of the `.is_Identity` method, which is testing exactly what I provided. However, I found it more efficient to not first create a `Permutation` instance.
But you are right with the 2nd concern, there even is an `is_trivial` property for `PermutationGroup`s, which is definitely a better solution.

Reply:
Is this identity test included somewhere?

Reply:
No, it is not, but it is not necessary. In ``PermutationGroup.__new__`` there is a check that removes all identity permutations if there is more than one argument/permutation given. Therefore, either only non-trivial permutations remain or there is only one argument that is the identity permutation. In the second case, ``is_trivial`` does its job.
This means it is impossible to create a ``PermutationGroup`` instance that contains the identity permutation in the list of generators other than the identity group.

However, there is another flaw, but it is not directly related to this issue: There is a somewhat undocumented ``dups=False`` kwarg which disables duplicate removal.
``PermutationGroup(Permutation(5), Permutation(5), dups=False)`` results in an error, but not related to normal subgroup testing.